Octokit/Models/Response/Commit.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.ObjectModel;
 using System.Diagnostics;
 using System.Linq;
+using Octokit.Models.Response;
```

Comment:
This shouldn't be necessary when the namespace is updated as per the other comment below...